docs/manual/cluster-admin/how-to-renew-k8s-cert.md

```diff
@@ -0,0 +1,129 @@
+# How to renew kubenetes certs
+
+The k8s API server's cert will expire every year, and will cause OpenPAI cluster not available.
+For more details, please refer to [Certificate Management with kubeadm](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/).
+
+## The alert from cert expiration checker
+
+If the admin setup the [cert expiration checker](./how-to-use-alert-system.md#Cluster-k8s-cert-expiration-checker) in alert manager, the admin email receiver will got the alert email before the cert expired.
+If the cert already expired, the user will got the error in web portal:
+![cert expired](./imgs/cert-expired.png)
+
+## Generate the new certificates and tokens
+
+### Generate the new certs on master node
+
+On the master node use the following commands generate the new certificates:
+
+```bash
+# On master - See https://kubernetes.io/docs/setup/certificates/#all-certificates
+sudo kubeadm alpha certs renew apiserver
+sudo kubeadm alpha certs renew apiserver-etcd-client
+sudo kubeadm alpha certs renew apiserver-kubelet-client
+sudo kubeadm alpha certs renew front-proxy-client
+```
+
+### Generate new kube-configs with embedded certificates
+
+On the master node use the following commands generate the new kube-configs:
+
+```bash
+sudo kubeadm alpha kubeconfig user --org system:masters --client-name kubernetes-admin  > admin.conf
+sudo kubeadm alpha kubeconfig user --client-name system:kube-controller-manager > controller-manager.conf
+sudo kubeadm alpha kubeconfig user --org system:nodes --client-name system:node:$(hostname) > kubelet.conf
+sudo kubeadm alpha kubeconfig user --client-name system:kube-scheduler > scheduler.conf
+
+# chown and chmod so they match existing files
+sudo chown root:root {admin,controller-manager,kubelet,scheduler}.conf
```

Comment:
this may cause issue if the user were not logged in as root

Reply:
Update the commands:
```bash
# please replace <user> to your current user name (e.g. root, core)
sudo chown <user> {admin,controller-manager,kubelet,scheduler}.conf
```